assets/icons/earth.svg

```diff
@@ -0,0 +1 @@
+<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 23 23"><defs><style>.a{fill:#fff;}</style></defs><path class="a" d="M11.5,23A11.513,11.513,0,0,1,0,11.5,11.368,11.368,0,0,1,.772,7.389a.771.771,0,0,1,.041-.116A11.54,11.54,0,0,1,4.973,2.041a11.453,11.453,0,0,1,13.944.686l.012,0a.071.071,0,0,1,.02.008.345.345,0,0,1,.061.064l.019.024A11.486,11.486,0,0,1,11.5,23ZM1.333,10.44h0c-.036.365-.054.722-.054,1.06a10.2,10.2,0,0,0,15.479,8.752l-.813-.925a.641.641,0,0,1-.118-.647l1.038-2.766L15.287,15.3c-1.039-.405-1.316-2.1-1.361-2.435a.649.649,0,0,1,.007-.208c.026-.131.662-3.207,2.648-4.115a6.834,6.834,0,0,1,2.837-.469,3.567,3.567,0,0,0,1.61-.24,10.162,10.162,0,0,0-2.374-3.617c-.883,1.123-2.575,3.049-3.97,3.322a2.367,2.367,0,0,1-.465.048,1.78,1.78,0,0,1-1.339-.559A1.8,1.8,0,0,1,12.524,5.3c.376-1.23.592-3.464.109-3.91a.1.1,0,0,1-.018-.025l0-.007A.058.058,0,0,0,12.6,1.34a10.163,10.163,0,0,0-5.242.824L8.4,2.506A2.1,2.1,0,0,1,9.823,4.2l.351,2.448a2.1,2.1,0,0,1-.664,1.842L7.88,9.976c-.673.61-.764.843-.772.915a2.822,2.822,0,0,0,.967.753c.413.254,1.951,1.031,2.412,1.263a2.489,2.489,0,0,1,.956,2.2,1.087,1.087,0,0,1-.6.974.787.787,0,0,0,.007.6,1.893,1.893,0,0,1,.626,1.36,1.366,1.366,0,0,1-.6,1.054,1.555,1.555,0,0,1-.926.337,1.941,1.941,0,0,1-1.189-.521L8.612,18.8l-2.426-1.77a2.1,2.1,0,0,1-.856-1.565l-.058-.986a.815.815,0,0,0-.485-.694,3.57,3.57,0,0,1-2.6-1.571A11.021,11.021,0,0,1,1.333,10.44ZM21.416,9.067a5.3,5.3,0,0,1-1.985.279,5.7,5.7,0,0,0-2.32.355c-1.052.48-1.69,2.172-1.9,3.1a2.35,2.35,0,0,0,.542,1.309l2.164.846a.641.641,0,0,1,.365.818l-1.124,3,.659.751a10.19,10.19,0,0,0,3.908-8.02A10.048,10.048,0,0,0,21.416,9.067ZM5.811,3.011A10.245,10.245,0,0,0,2.022,7.689a8.041,8.041,0,0,0,1.264,3.858,2.324,2.324,0,0,0,1.685.968.641.641,0,0,1,.176.034,2.083,2.083,0,0,1,1.4,1.847l.06.989a.806.806,0,0,0,.332.609l2.619,1.921c.216.174.306.236.382.236.054,0,.1-.032.171-.086.092-.15-.035-.371-.192-.512a1.82,1.82,0,0,1-.349-1.635,1.544,1.544,0,0,1,.6-.928,1.279,1.279,0,0,0-.333-.984c-.116-.051-1.909-.95-2.348-1.22l-.012-.007c-.606-.37-1.436-.876-1.617-1.622-.217-.887.694-1.71,1.132-2.106L8.648,7.546a.816.816,0,0,0,.258-.718l-.349-2.45A.816.816,0,0,0,8,3.718L7.63,3.6l-.013,0L7.4,3.52C6.853,3.344,6.294,3.163,5.811,3.011ZM14.09,1.623a10.253,10.253,0,0,1-.343,4.046.517.517,0,0,0,.081.495.493.493,0,0,0,.385.14,1.229,1.229,0,0,0,.224-.023c.856-.169,2.283-1.669,3.245-2.9A10.114,10.114,0,0,0,14.09,1.623Z"/></svg>
```

Comment:
Please do not fill the icons for easier coloring, I usually edit in AI with registration color, see pic below, or you may remove the `<defs />`

![image](https://user-images.githubusercontent.com/17264716/41279705-d162e22c-6e5f-11e8-810f-cd582e97b217.png)